.drone.yml

```diff
@@ -1,17 +0,0 @@
----
-
-kind: pipeline
-type: kubernetes
-name: lint
-
-steps:
-- name: lint docker-compose
-  image: docker/compose
-  commands:
-    - docker-compose -f docker/docker-compose.yml config
```

Comment:
Because we use the centralised config now this linting step happens on that project, its not lost.